CMake/cdat_modules_extra/python_configure_step.cmake.in

```diff
@@ -29,7 +29,7 @@ if(APPLE)
 endif()
 
 if(APPLE)
-  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
+  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --with-system-expat --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
```

Comment:
Yep - see https://travis-ci.org/PCMDI/pcmdi_metrics/jobs/89462940#L27-L39 and expand the print statements which query available libraries before calling cmake